app/style/main.scss

```diff
@@ -30,7 +58,6 @@ $duration: .2s;
     outline: inherit;
   }
 
-  // Will be removed
   .xb-cookie-kit__refresh-button {
```

Comment:
I didn't even think about the general button styles having a higher specificity than the BEM compliant selector I suggested. But in addition to your suggestions of using `!important` and the selector without the `&`, there is another option. It is a hack. Instead of using `.xb-cookie-kit__refresh-button` which compiles to `.xb-cookie-kit .xb-cookie-kit__refresh-button` because of the nesting you can double up with the same class name. So, if you use something that compiles to `.xb-cookie-kit__refresh-button.xb-cookie-kit__refresh-button` (note that there is no space between the two class selectors), then it will work too because its specificity is 0.2.0 even though it has duplicate class selectors. I am not sure if `&__refresh-button.&__refresh-button` will work in SASS.

Note: I am not suggesting we use this technique _here_, but it is sometimes useful when you need to override a third-party's styles.

I am going to approve this for now since I am going to hop on this project and make some changes and I would like to have these changes with my upcoming changes.

But the BEM way to fix this is to not have the nested `button` type selector. Instead, change it to `&__button` which will compile to `.xb-cookie-kit__button`. Then add this class to the HTML along with `xb-cookie-kit__refresh-button`. Then both rule sets will have a selector with a specificity of 0.1.0 and because the refresh button's style follow the general button styles, the cascade will take effect.

I hope these long comments make sense. You may need to read them a few times for them to set in.

Thanks for reading all this.

Reply:
@maklyakov You're welcome. I am always happy to help a fellow developer.